src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -192,6 +201,10 @@ public static function getContacts(array $owner, array $rel, string $module, int
 		}
 
 		if (!$show_contacts) {
+			if (!empty($cachekey)) {
```

Comment:
According to the description of the parameter, I believe you should have checked the value of `$nocache` here.

Reply:
No. The parameter controls whether the cached value should be used or not. It is intentional that the cache is updated under any condition.

Reply:
Very well, but this is not what the description of the parameter suggests.